xrootd/protocol/protocol/protocol.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Package protocol contains the structures describing request and response
+for protocol request (see XRootD specification).
+
+Response consist of 3 parts:
+
+1) GeneralResponse - general response that is always returned and specifies protocol version and flags describing server type.
+
+2) SecurityInfo - a response part that is added to general response
+if `ReturnSecurityRequirements` is provided and server supports it.
+It contains security version, security options, security level,
```

Comment:
`s/security version/the security version/`
`s/security options/the security options/`
`s/security level/the security level/`